projects/kit/components/input-count/input-count-options.ts

```diff
@@ -0,0 +1,42 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface InputCountOptions {
+    readonly icons: Readonly<{
+        up: PolymorpheusContent<TuiContextWithImplicit<TuiSizeL>>;
+        down: PolymorpheusContent<TuiContextWithImplicit<TuiSizeL>>;
+    }>;
+    appearance: string;
+    hideButtons: boolean;
+    min: number;
+    max: number;
+    step: number;
+    postfix: string;
+}
+
+// TODO: remove in ivy compilation
+export const PASSWORD_ICON_UP = 'tuiIconPlus';
+export const PASSWORD_ICON_DOWN = 'tuiIconMinus';
+
+/** Default values for the input password options. */
+export const TUI_INPUT_COUNT_DEFAULT_OPTIONS: InputCountOptions = {
+    icons: {
+        up: PASSWORD_ICON_UP,
+        down: PASSWORD_ICON_DOWN,
+    },
+    appearance: 'textfield',
+    hideButtons: false,
```

Comment:
Right now we force hide buttons in tables by comparing appearance to table. That's rather hardcode. @vladimirpotekhin  @nsbarsukov what do you think, it probably makes sense to alter this token in `Table` directive now? With a factory, skipSelf and spread with `hideButtons` set to `true`? Sounds like that would be more idiomatic.

Reply:
In this case someone can rewrite token if they use nested components inside the table. Or am I wrong?

Reply:
In this case, if somebody uses component inside table and for some reason want to change it, they can create a directive that would override it, yes. I don't think that's bad.

Reply:
Ok. I've already done this as an example. @vladimirpotekhin @nsbarsukov you can check it right here

Reply:
@waterplea there is also tuiTableMode directive with TUI_TEXTFIELD_APPEARANCE token. Should we add TUI_INPUT_COUNT_OPTIONS here as well? In this case, should we move TUI_INPUT_COUNT_OPTIONS into core package?

Reply:
No, that directive was there before we introduced table component. I believe we might end up just dropping it in the next major release.

Reply:
Oops, @KarimovDev I see this directive is still actually in use on the demo and that fails the screenshots. Let's copy-paste this factory for the time being then.

Reply:
Done